tox.ini

```diff
@@ -3,56 +3,39 @@ name = tmd
 testdeps =
     nose
     tqdm
+commands = nosetests
+
+[testenv]
+deps = {[base]testdeps}
+extras = viewer
+commands = nosetests
 
 [tox]
 envlist =
-    check-version
     pycodestyle
-    pylint
+    lint
     py27
-    py35
-
-[testenv:check-version]
-skip_install = true
-# commands = /gpfs/bbp.cscs.ch/apps/viz/platform/binfiles/bbp-nse-ci/0.0.1/bin/do_release.py -p . check-version
-
-
-[testenv:pycodestyle]
-deps = pycodestyle
-commands = true | pycodestyle {[base]name} --exclude tests
-
-[testenv:pylint]
-deps = pylint
-commands = true | pylint {[base]name}
-
-[testenv:py27]
-deps = {[base]testdeps}
-commands = true | nosetests
+    py36
 
-[testenv:py35]
-deps = {[base]testdeps}
-commands = true | nosetests
-
-[testenv:py36]
-deps = {[base]testdeps}
-commands = true | nosetests
+[testenv:lint]
+basepython = python3
+deps =
+    pycodestyle
+    pylint
+commands =
+    pycodestyle --config=pycodestylerc --exclude=tests tmd
+    pylint --rcfile=pylintrc --extension-pkg-whitelist=numpy --ignore=tests tmd
 
 [testenv:coverage]
 deps =
     {[base]testdeps}
     coverage
 commands =
-    true | coverage run --source {[base]name} {envbindir}/nosetests
-    true | coverage report --show-missing
-    true | coverage xml
-
-[testenv:docs]
-changedir = doc
-deps = sphinx
-commands = true | make html
-whitelist_externals = make
+    coverage run --source {[base]name} {envbindir}/nosetests
+    coverage report --show-missing
+    coverage xml
 
 # E731: do not assign a lambda expression, use a def
 [pycodestyle]
-ignore = E731
+ignore = E731,W504
```

Comment:
Can you please add a comment for W504? I think it's the line continuation, but better to doc.